tests/Integration/Behaviour/Features/Scenario/Profile/profile_permissions.feature

```diff
@@ -109,35 +109,35 @@ Feature: Manage profile permissions from BO
 
   Scenario: I can get profile module permissions for profiles
     Then profile super_admin should have the following permissions for modules:
-      | ps_banner      | view,configure,uninstall |
-      | ps_wirepayment | view,configure,uninstall |
+      | ps_banner            | view,configure,uninstall |
+      | ps_emailsubscription | view,configure,uninstall |
```

Comment:
Changed module to another one because now we are using test modules and ps_wirepayment doesnt exist there. So i just took another random one.